sui_types/src/batch.rs

```diff
@@ -35,8 +48,11 @@ pub struct AuthorityBatch {
     /// The digest of the previous block, if there is one
     pub previous_digest: Option<BatchDigest>,
 
-    // The digest of all transactions digests in this batch
+    /// The digest of all transactions digests in this batch
     pub transactions_digest: [u8; 32],
+
+    /// The set of transactions in this batch
+    pub transaction_batch: TransactionBatch,
```

Comment:
Ok, this is a biggie -- why are we now including the set of transactions into the batch, as well as disparate transactions? This now means that we trasmit them twice, and also that batches are no more constant-ish size. The batch structure was specifically designed to only contain meta-data to avoid this issue.